src/App.css

```diff
@@ -14,7 +14,7 @@
 section {
   max-width: 1280px;
   margin: 0 auto;
-  padding: 0.5rem 0.5rem 2rem 0.5rem;
+  padding: 0.5rem;
```

Comment:
I personally think that it is great to have larger bottom padding because it looks a little bit weird and uncomfortable if the contents are full up to the bottom